packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -104,14 +197,22 @@ export const OdieSendMessageButton = () => {
 						inputRef={ inputRef }
 						setSubmitDisabled={ setSubmitDisabled }
 						keyUpHandle={ onKeyUp }
+						onPasteHandle={ onPaste }
 					/>
 					{ isChatBusy && <Spinner className="odie-send-message-input-spinner" /> }
-					<AttachmentButton attachmentButtonRef={ attachmentButtonRef } />
+					{ showAttachmentButton && (
+						<AttachmentButton
+							attachmentButtonRef={ attachmentButtonRef }
+							onFileUpload={ handleFileUpload }
+							isAttachingFile={ isAttachingFile }
+						/>
+					) }
 					<button type="submit" className={ buttonClasses } disabled={ submitDisabled }>
 						<SendMessageIcon />
 					</button>
 				</form>
 			</div>
+			{ showAttachmentButton && <DropZone onFilesDrop={ onFilesDrop } /> }
```

Comment:
Let's pass a label to dropzone so it's accurate, currently it says "Drop files to upload". It should say something like "Drop here to share the image with our happiness engineers". 
```suggestion
			{ showAttachmentButton && <DropZone onFilesDrop={ onFilesDrop } label={ __( 'Drop files to upload". It should say something like "Share this image with our happiness engineers' ) }  /> }
```